### PR TITLE
Updated PTZ preset functionality

### PR DESCRIPTION
**Pull request by Gartom**
- Updated PTZ presets to full range available in cameras (0-255).
- Updated companion presets for PTZ presets.
- Fixed a bug for companion presets for PTZ presets when PTZ preset number in HEX contained the letter A to F.

### CODE REVIEW

**Comment by Gartom** [comment]
Closes #13, #14 and #15.

**Review by JeffreyDavidsz — COMMENTED** [review]
Does preset 0 exists? Seems odd for a camera

**Comment by Gartom** [comment]
Preset 0 is a special preset that is used as "home" position after startup if it has been set.

**Comment by Gartom** [comment]
Just realized that the bugfix I made is not backwards compatible with old config files and need to be made slightly different. Don't merge this yet...

File: index.js [hunk 1]
```diff
@@ -1220,7 +1220,7 @@ instance.prototype.action = function(action) {
 			break;
 
 		case 'speedPset':
-			cmd ='\x81\x01\x7E\x01\x0B' + String.fromCharCode(parseInt(opt.val,16) & 0xFF) + String.fromCharCode(parseInt(opt.speed,16) & 0xFF) + '\xFF';
+			cmd ='\x81\x01\x06\x01' + String.fromCharCode(parseInt(opt.val,16) & 0xFF) + String.fromCharCode(parseInt(opt.speed,16) & 0xFF) + '\xFF';
```
Comment (astellingwerf) [inline index.js:1223]: What does this have to do with the rest of the changeset?

Comment (Gartom) [reply index.js:1223]: Maybe it should have been added in a separate PR but it is the fix for bug #13 as mentioned in a comment above. It is the actual command to set the "Preset Recall Speed" in the PTZ camera, and it becomes relevant when using the preset functionality that is updated in this PR.

From the ["PTZOptics VISCA over IP Commands"](https://ptzoptics.com/wp-content/uploads/2020/08/PTZOptics-VISCA-over-IP-Rev-1_2-8-20.pdf) document:
bild

**Comment by krocheck** [comment]
This needs to be revisited, I believe.  Although the command was updated as reported by the ticket, [PTZOptics documentation](https://ptzoptics.com/wp-content/uploads/2020/11/PTZOptics-VISCA-over-IP-Rev-1_2-8-20.pdf) doesn't show support for a preset number, only the speed option.

image

**Comment by Gartom** [comment]
The PTZOptics user manual is not complete regarding presets.

Therefore, I based the update on responses from PTZOptics in their forum:

https://help.ptzoptics.com/support/discussions/topics/13000022947/page/2

Colin at PTZOptics referred to this page for details about preset numbers, which corresponds to the implementation:
https://conferenceroomsystems.freshdesk.com/support/solutions/articles/13000077943-ptzoptics-preset-commands-reserved-presets-and-special-functions

**Comment by Gartom** [comment]
@krocheck , is there still an issue?
